source/mongodump.txt

```diff
@@ -22,17 +22,19 @@
 .. |tool-binary| replace:: :binary:`~bin.mongodump`
 .. |mongodb-aws-example| replace:: :ref:`mongodump-example-connect-using-aws-iam`
 
-Synopsis
---------
-
 :binary:`~bin.mongodump` is a utility that creates a binary export of
-a database's contents.
-
-:binary:`~bin.mongodump` can export data from:
+a database's contents. :binary:`~bin.mongodump` can export data from:
 
 - Standalone deployments
 - Replica sets
 - Sharded clusters
+- :atlas:`Serverless instances 
+  </create-database-deployment/#serverless-instances>` running a
+  non-rapid-release version of MongoDB
```

Comment:
This can just say serverless instances. There are no serverless instances running non-rapid-release versions